### ISSUE TITLE
VST effects won't load in 1.3.0

### ISSUE THREAD

**Issue by Sargeanthost** [LMMS/lmms#6094]
### Bug Summary
Version 1.3.0 alpha 1.102 doesn't support VST effects for some reason.
#### Steps to reproduce
Open track > effects > add new > click a 3rd party vst (e.g. backmask)

You will notice it instantly populates an effect tab and takes no time to load it in. Also, the `Controls` button is missing.
#### Expected behavior
How it works in 1.2.2 where it loads into LMMS.
#### Actual behavior
No button, empty tab.
#### Screenshot
![image](https://user-images.githubusercontent.com/36427968/126026462-5df21dcc-eac3-462a-b21c-20c306e49d4c.png)

#### Affected LMMS versions
latest nightly build on the website as of this posting,  1.3.0 alpha 1.102

**Comment by Spekular** [comment]
Can you provide your operating system as well? I don't experience this issue on Windows (10).

**Comment by Sargeanthost** [comment]
Windows 10 as well

**Comment by sdasda7777** [comment]
I'm not experiencing this issue either. Can you reproduce it with any VST effect?

**Comment by Sargeanthost** [comment]
any vst I try except one, I can publish a short youtube clip if that would help.

**Comment by Sargeanthost** [comment]
I also saw somewhere to install as administrator and I remember clicking yes on that prompt, so I doubt its that same issue.

**Comment by sdasda7777** [comment]
> any vst I try except one, I can publish a short youtube clip if that would help.

Hard to say for sure, but if it's not much bother for you, go for it

**Comment by GingkathFox** [comment]
Reproduced on Arch with latest master, logs below:

```
kk003c:err:service:process_send_command service protocol error - failed to write pipe!
003c:fixme:service:scmdatabase_autostart_services Auto-start service L"Bonjour Service" failed to start: 1053
0070:err:ntdll:RtlpWaitForCriticalSection section 000000007BC61540 "../wine/dlls/ntdll/loader.c: loader_section" wait timed out in thread 0070, blocked by 0078, retrying (60 sec)
003c:err:service:process_send_command receiving command result timed out
003c:fixme:service:scmdatabase_autostart_services Auto-start service L"MountMgr" failed to start: 1053
Remote plugin did not connect.
Error while writing.
QProcess: Destroyed while process ("/home/home/gingka/Builds/lmms/build/plugins/RemoteVstPlugin64") is still running.
Process error:  QProcess::Crashed
Remote plugin crashed
003c:err:service:process_send_command receiving command result timed out
003c:fixme:service:scmdatabase_autostart_services Auto-start service L"winebus" failed to start: 1053
00a4:fixme:ntoskrnl:IoRegisterShutdownNotification stub: 000000000024A9F0
003c:err:service:process_send_command service protocol error - failed to write pipe!
003c:fixme:service:scmdatabase_autostart_services Auto-start service L"PaceLicenseDServices" failed to start: 1053
wine: Unhandled page fault on read access to 0000000000000040 at address 00000001400036EB (thread 003c), starting debugger...
Remote plugin did not connect.
Error while writing.
QProcess: Destroyed while process ("/home/home/gingka/Builds/lmms/build/plugins/RemoteVstPlugin64") is still running.
Process error:  QProcess::Crashed
Remote plugin crashed
```
it also affects vst instruments.

**Comment by Sargeanthost** [comment]
> > any vst I try except one, I can publish a short youtube clip if that would help.
> 
> Hard to say for sure, but if it's not much bother for you, go for it

Sorry for the wait, forgot about this thread, I'll edit this with a clip tomorrow.

**Comment by Sargeanthost** [comment]
I realized a clip won't do much since there's nothing to show, but I do have some more useful information. I've found some VST's that work such as MDA Piano and 4L2 and Blue Cat knobs.
